src/main/java/mate/academy/service/MovieService.java

```diff
@@ -9,4 +9,8 @@
     Movie get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` or throwing an exception if the deletion fails. Returning a Boolean might not be the best approach for error handling.